index.html

```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
-            <li>javaScript</li>
+    <section class="work-three work">
+        <img class ="img-work img-work-three" src="img/Portfolio3.svg" alt="Tonic">
+      <div class="body-content">   
+        <h2>Tonic</h2>
+          <ul class="work-top">
+            <li class="canopy">CANOPY</li>
+            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+            <li class="dev">back end dev</li>
+            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+            <li class="dev">2015</li>
           </ul>
-        <form action="#">
-            <button class="btn-body" type="submit" name="submit">see project</button>
-        </form>
+          <p class="body-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+            <ul class="work-bottom">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          <form action="#">
+              <button class="btn-body btn" type="submit" name="submit">see project</button>
+          </form>
+        </div>
     </section>
     <!-- Third section ends -->
     <!-- Fourth section starts -->
-    <section class="work-one work">
-      <img class ="img-work" src="img/Portfolio4.svg" alt="Tonic">
-      <h2>Tonic</h2>
-        <ul class="work-top">
-          <li class="canopy">CANOPY</li>
-          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
-          <li class="dev">back end dev</li>
-          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
-          <li class="dev">2015</li>
-        </ul>
-        <p class="body-paragraph">
-          A daily selection of privately personalized reads; no accounts or sign-ups required.
-        </p>
-          <ul class="work-bottom">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
+    <section class="work-four work">
+        <img class ="img-work img-work-four" src="img/Portfolio4.svg" alt="Tonic">
+        <div class="body-content">
+          <h2>Multi-Post Stories</h2>
+          <ul class="work-top">
+            <li class="canopy">CANOPY</li>
+            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+            <li class="dev">back end dev</li>
+            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+            <li class="dev">2015</li>
           </ul>
-        <form action="#">
-            <button class="btn-body" type="submit" name="submit">see project</button>
-        </form>
+          <p class="body-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+              <ul class="work-bottom work-bottom1">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <ul class="work-bottom2">
```

Comment:
The height of the buttons is a little bit large on the tablet view. Please see the design and adjust the height. Kindly see the below screenshot for reference.
![buttons](https://user-images.githubusercontent.com/35504149/130109652-5818138e-4f39-40d4-8c92-25c888d75f37.png)